### PR TITLE
Php sample (it is a moving outside of the repo)

### PR DESCRIPTION
**Pull request by ksvirkou-hubspot**

### CODE REVIEW

**Review by atanasiuk-hubspot — APPROVED** [review]

File: php/composer.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "minimum-stability": "stable",
+
+    "require": {
+        "hubspot/api-client": "*",
```
Comment (atanasiuk-hubspot) [inline php/composer.json]: Should we fix the specific api-client version?

**Review by atanasiuk-hubspot — CHANGES_REQUESTED** [review]
Please fix the SDK version

File: php/composer.json (hunk 1 above)
Comment (ksvirkou-hubspot) [reply php/composer.json]: I  think we shouldn't because it is fix in composer.lock

File: php/composer.lock [hunk 2]
```diff
@@ -0,0 +1,907 @@
...
+            ],
+            "support": {
+                "issues": "https://github.com/guzzle/psr7/issues",
+                "source": "https://github.com/guzzle/psr7/tree/1.8.2"
+            },
+            "time": "2021-04-26T09:17:50+00:00"
+        },
+        {
+            "name": "hubspot/api-client",
+            "version": "4.0.0",
```
Comment (ksvirkou-hubspot) [inline php/composer.lock:244]: pay attention please @atanasiuk-hubspot

**Review by atanasiuk-hubspot — APPROVED** [review]

**Pull request merged**